### PR TITLE
Feature/325 fe styleguide

### PR DESCRIPTION
**Pull request by er-ant**

### CODE REVIEW

**Comment by er-ant** [comment]
[(3) Styleguide for traceto](https://trello.com/c/G01g42Qg/325-3-styleguide-for-traceto)

File: ts/README.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+  <a name="imports--global-imports"></a><a name="6.1"></a>
+  - [6.1](#imports--global-imports) Avoid global imports from library, specific modules imports are more preferable:
+    ```ts
+    // bad
+    import { MatInputModule } from '@angular/material';
+    import { Observable } from 'rxjs/Rx';
+
+    // good
+    import { MatInputModule } from '@angular/material/input';
+    import { Observable } from 'rxjs';
```
Comment (tracetoio-jingyi) [inline ts/README.md:297]: The observable import example is a bit confusing here, please remove or change to another one.

**Review by tracetoio-jingyi — CHANGES_REQUESTED** [review]

File: ts/README.md (hunk 1 above)
Comment (er-ant) [reply ts/README.md:297]: @tracetoio-jingyi updated, please check.

**Review by tracetoio-weilong — APPROVED** [review]

**Review by tracetoio-jingyi — APPROVED** [review]

File: scss/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Traceto.io TypeScript Styleguide
+For styles in our projects we use BEM conventions, with next basic rules:
```
Comment (traceto) [inline scss/README.md]:
```suggestion
For styles in our projects we use [BEM](http://getbem.com/introduction/) conventions, with next basic rules:
```

File: ts/README.md [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+      const resultObject: IBarResult = <IBarResult>{
+        array: includeArray,
+        included: aIncluded
+      };
+
+      return resultObject;
+    }
+    ```
+  <a name="type--fullness"></a><a name="1.2"></a>
+  - [1.2](#type--fullness) If you do not know or not sure about type of any entity in your code, assign type `any`:
```
Comment (traceto) [inline ts/README.md]: Just recheck this if there is any

File: ts/README.md [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+    1. Private properties
+    1. Public properties
+    1. Constructor
+    1. Framework's methods (for example ngOnInit, if Angular is used)
+    1. Private methods
+    1. Public methods
+    1. Template methods (if frontend)
+
+  <a name="classes--constructor"></a><a name="3.2"></a>
+  - [3.2](#classes--constructor) Constructor requirments:
```
Comment (traceto) [inline ts/README.md]: typo for requirements

**Review by traceto — APPROVED** [review]

**Pull request merged**